ui/app/com/gu/recipeasy/views/admin/index.scala.html

```diff
@@ -4,6 +4,7 @@
     <h2>Recipeasy Admin Area</h2>
     <ul>
         <li><a href="@routes.Application.recentActivity">Recent activity</li>
+        <li><a href="@routes.Application.recentUserActivity("alice.skywalker@guardian.co.uk")">Recent activity of a specific user</li>
```

Comment:
So......... I want the url to appear on the admin landing page (so that we can quickly remember what it is), but without singling out a single user. "Alice Skywalker" is not a real user (not a real employee), so her activity is empty.

I did this because otherwise a link from the curation page would open this new admin page, but I don't want users to even know about the existence of the page.

The intended use of this page is @alastairjardine click on it and manually replace the fake email address with the email address of the user he is interested in. 

Alternatively, I can add a link on the curation page, but I will have to make that link appear only to the admins who actually can see that page.

Either way works for me :)